bin/GameIndex.yaml

```diff
@@ -18376,6 +18379,9 @@ SLPM-60265:
 SLPM-60273:
   name: "Zettai Zetsumei Toshi 2: Itetsuita Kioku-tachi [Trial B]"
   region: "NTSC-J"
+SLPM-60282:
+  name: "Lucky * Star RAvish Romance  [Trial]"
```

Comment:
Possible typo here. Shouldn't have two spaces between Romance and [Trial].

Reply:
Done.